src/web_asset_source.rs

```diff
@@ -114,7 +114,7 @@ async fn get<'a>(path: PathBuf) -> Result<Box<Reader<'a>>, AssetReaderError> {
         )
     })?;
 
-    let client = surf::Client::new().with(surf::middleware::Redirect::default());
```

Comment:
Was this change intentional?

Reply:
yeah, when `surf::middleware::Redirect` is added, `web_image` example errors with something like:
```
ERROR bevy_asset::server: Encountered an I/O error while loading asset: unexpected status code 500 while loading https://cdn.pixabay.com/photo/2015/03/10/17/23/youtube-667451_1280.png: Head byte length should be less than 8kb
```

Reply:
This was added in a recent PR, @soraxas what do you think? Won't this break redirecting for you?

Reply:
related surf issue: https://github.com/http-rs/surf/issues/289

Reply:
I have encountered status code 500 but that was due to bot prevention arise from the meta check; after disabling meta check I have not had that issue. 

I'm currently away from my machine; I can test our the PR later